<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:black</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>120</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#backButton{
	border-width: 2px;
	border-style: none;
	border-color: white;
	border-radius: 10px;
	background-color: rgba(110, 126, 141, 255);
	color: white;
}

QPushButton#backButton:pressed{
	padding-left: 2px;
	padding-top: 2px;
	background-color: rgba(110, 126, 141, 200);
}
</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QComboBox" name="stationPicker">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>440</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI Semilight</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox#stationPicker{
	color: rgba(163, 122, 122, 255);
	background-color: white;
	selection-background-color: transparent;
	border: none;
	border-radius: 5px;
}

QComboBox:drop-down {
	border-width: 0px;
}

QComboBox:down-arrow{
	image: url(noimg); 
	border-width: 0px;
}

QComboBoxr:on{ 
	background: white;
}

QListView{ 
	background-color: white;
	color: rgba(122, 163, 122, 255);
}</string>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>200</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgba(122, 163, 122, 255);
	color: white;
	border-style: solid;
	border-color: white;
	border-radius: 10px;
	border-width: 2px;
}

QPushButton:pressed{
	padding-left: 5px;
	padding-top: 5px;
	background-color: : rgba(122, 163, 122, 200);
}
</string>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>200</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgba(110, 126, 141, 255);
	color: white;
	border-style: solid;
	border-color: white;
	border-radius: 10px;
	border-width: 2px;
}

QPushButton:pressed{
	padding-left: 5px;
	padding-top: 5px;
	background-color:  rgba(110, 126, 141, 200);
}
</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="bar1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>30</width>
      <height>301</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgba(122, 163, 122, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar2">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>350</y>
      <width>30</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgba(122, 163, 122, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar3">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>350</y>
      <width>30</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgba(122, 163, 122, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar4">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>350</y>
      <width>30</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgba(122, 163, 122, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar5">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>350</y>
      <width>30</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgba(122, 163, 122, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar6">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>350</y>
      <width>30</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgba(122, 163, 122, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar7">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>350</y>
      <width>30</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgba(122, 163, 122, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar8">
    <property name="geometry">
     <rect>
      <x>284</x>
      <y>350</y>
      <width>30</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgba(122, 163, 122, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar9">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>350</y>
      <width>30</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgba(122, 163, 122, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar10">
    <property name="geometry">
     <rect>
      <x>354</x>
      <y>350</y>
      <width>30</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgba(122, 163, 122, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar11">
    <property name="geometry">
     <rect>
      <x>389</x>
      <y>350</y>
      <width>31</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgba(122, 163, 122, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar12">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>350</y>
      <width>30</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgba(122, 163, 122, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar13">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>350</y>
      <width>30</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgba(122, 163, 122, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar14">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>350</y>
      <width>30</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgba(122, 163, 122, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar15">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>350</y>
      <width>30</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgba(122, 163, 122, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar16">
    <property name="geometry">
     <rect>
      <x>565</x>
      <y>350</y>
      <width>30</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgba(122, 163, 122, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar17">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>350</y>
      <width>30</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgba(122, 163, 122, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar18">
    <property name="geometry">
     <rect>
      <x>635</x>
      <y>350</y>
      <width>30</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgba(122, 163, 122, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar19">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>350</y>
      <width>30</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgba(122, 163, 122, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar20">
    <property name="geometry">
     <rect>
      <x>705</x>
      <y>350</y>
      <width>30</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgba(122, 163, 122, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bar21">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>350</y>
      <width>30</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgba(122, 163, 122, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="visPicker">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>135</y>
      <width>200</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI Semilight</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
	color: rgba(163, 122, 122, 255);
	background-color: white;
	selection-background-color: transparent;
	border: none;
	border-radius: 5px;
}

QComboBox:drop-down {
	border-width: 0px;
}

QComboBox:down-arrow{
	image: url(noimg); 
	border-width: 0px;
}

QComboBoxr:on{ 
	background: white;
}

QListView{ 
	background-color: white;
	color: rgba(122, 163, 122, 255);
}</string>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
